README.md

```diff
@@ -518,6 +518,25 @@ The `<...>` notation means the argument.
 * `irb`
   * Invoke `irb` on the current frame.
 
+### Trace
+
+* `trace`
+  * Show available tracers list.
+* `trace line`
+  * Add a line tracer. It indicates line events.
+* `trace call`
```

Comment:
can `trace call` also take an optional expression? with that it'll only trace calls made from the expression's value.

Reply:
I can't catch enough.
could you give me an example?

Reply:
```ruby
p1 = Post.find(1)
p2 = Post.find(2)
binding.b(do: "trace call p1")

p1.update!(title: "bar") #=> this and other internal calls on p1 will be printed
p2.update!(title: "foo") #=> this will not be printed

a_method #=> this will not be printed
```

Reply:
`trace pass p1` doesn't help?

Reply:
ah sorry I thought `trace pass <expr>` only prints when the object is passed but not when it's called.
I'm happy with the `trace` command's functionalities then 👍

Reply:
This command is a bit confusing, but we can imagine that the receiver is the first parameter for the method.